### PR TITLE
added more information about DAY_PERIOD

### PR DESCRIPTION
**Pull request by nibinqtl**

### CODE REVIEW

File: zh/docs/edge-logic/supported-directives.md [hunk 1]
```diff
@@ -277,7 +277,7 @@ location @try_origin2 {
 | 字符串<br>修改 | TO_UPPER | ```eval_func $output TO_UPPER $input;```<br>把输入字符串转成大写。|
 | 字符串<br>修改 | TO_LOWER | ```eval_func $output TO_LOWER $input;```<br>把输入字符串转成小写。|
 | 字符串<br>修改 | SUBSTR | ```eval_func $output SUBSTR <start> <length> $input;```<br>获取输入字符串的一个子串，长度为```<length>```，起始位置为```<start>```。```<start>```可以是一个负数，就像Javascript的[substr()](https://www.w3schools.com/jsref/jsref_substr.asp)函数一样.|
-| 一日之内<br>的时间| DAY_PERIOD| ```eval_func $out DAY_PERIOD 19:00+0800 12h CN-night;```<br>如果时间在19:00+0800之后的12小时内，则返回'CN-night'|
+| 一日之内<br>的时间| DAY_PERIOD| ```eval_func $out DAY_PERIOD 19:00+0800 12h CN-night;```<br>如果时间在19:00+0800之后的12小时内，则返回'CN-night'，否则返回空字符串|
```
Comment (Jianghai-Quantil) [inline zh/docs/edge-logic/supported-directives.md:280]: An ending period is missing. We'd better add this to be consistent with other paragraphs.

File: zh/docs/edge-logic/supported-directives.md [hunk 2]
```diff
@@ -277,7 +277,7 @@ location @try_origin2 {
 | 字符串<br>修改 | TO_UPPER | ```eval_func $output TO_UPPER $input;```<br>把输入字符串转成大写。|
 | 字符串<br>修改 | TO_LOWER | ```eval_func $output TO_LOWER $input;```<br>把输入字符串转成小写。|
 | 字符串<br>修改 | SUBSTR | ```eval_func $output SUBSTR <start> <length> $input;```<br>获取输入字符串的一个子串，长度为```<length>```，起始位置为```<start>```。```<start>```可以是一个负数，就像Javascript的[substr()](https://www.w3schools.com/jsref/jsref_substr.asp)函数一样.|
```
Comment (Jianghai-Quantil) [inline zh/docs/edge-logic/supported-directives.md:279]: I suggest changing the ending period to ‘。’ to be consistent with other paragraphs.

**Review by Jianghai-Quantil — CHANGES_REQUESTED** [review]
There are 2 minor issues related to punctuation. The texts look good to me.

File: zh/docs/edge-logic/supported-directives.md (hunk 2 above)
Comment (nibinqtl) [reply zh/docs/edge-logic/supported-directives.md]: updated

**Review by Jianghai-Quantil — APPROVED** [review]

**Comment by Jianghai-Quantil** [comment]
Changes reviewed. Look good to me now. Changes can be merged.

**Pull request merged**